src/Bugsnag/Payload/Report.cs

```diff
@@ -7,7 +7,7 @@ namespace Bugsnag.Payload
 {
   public class Report : Dictionary<string, object>, IPayload
   {
-    private static readonly string _payloadVersion = "4";
```

Comment:
According to Mike Bull "4" is the preferred way so we should leave this as is and update the mazerunner repo to handle both.